Packs/DelineaSS/Integrations/DelineaSS/DelineaSS.py

```diff
@@ -478,8 +478,28 @@ def secret_rpc_changepassword_command(client, secret_id: str = '', newpassword:
     )
 
 
+def get_credentials(client,secret_id):
+    obj={}
+    secret = client.getSecret(secret_id)
+    items = secret.get('items')
+    for item in items:
+        if item.get('fieldName') == 'Username':
+            username = item.get('itemValue')
+        if item.get('fieldName') == 'Password':

```

Comment:
Why does the field `fieldName` in item will be both (Username and Password)? Should the second one (for password) be `itemValue` as in line 489?

Reply:
It is to extract value from the dictionary item.

Reply:
Yeah I got that, but you're extracting the field `fieldName` twice, in line 488 for `Username` and in line 490 for `Password`

Reply:
Yes. The fieldname is used to store the name of the field example username and password and their value is stored in itemvalue field. I want to extract both values. We have input like below : 
'items': [
            {
                'itemId': 14,
                'fileAttachmentId': None,
                'filename': None,
                'itemValue': 'user',
                'fieldId': 111,
                'fieldName': 'Username',
                'slug': 'username',
                'fieldDescription': 'The Unix Machine Username.',
                'isFile': False,
                'isNotes': False,
                'isPassword': False
            },
            {
                'itemId': 15,
                'fileAttachmentId': None,
                'filename': None,
                'itemValue': 'password',
                'fieldId': 110,
                'fieldName': 'Password',
                'slug': 'password',
                'fieldDescription': 'The password of the Unix Machine.',
                'isFile': False,
                'isNotes': False,
                'isPassword': True
            }
        ]